test/Microsoft.NET.Sdk.Functions.Generator.Tests/FunctionJsonConverterTests.cs

```diff
@@ -46,24 +66,144 @@ public class FunctionsClass
             public static void Run8() { }
         }
 
+        public class BindingAssertionItem
+        {
+            public string FunctionName { set; get; }
+
+            // key is binding type, value is binding parameter name.
+            public Dictionary<string, string> Bindings { set; get; }
+        }
+
+        public class BindingTestData : IEnumerable<object[]>
+        {
+            public IEnumerator<object[]> GetEnumerator()
+            {
+                yield return new object[] {
+                    new BindingAssertionItem
+                    {
+                        FunctionName="MyHttpTrigger",
+                        Bindings= new Dictionary<string, string>
+                        {
+                            {"httpTrigger", "request"}
+                        }
+                    }
+                };
+                yield return new object[] {
+                    new BindingAssertionItem
+                    {
+                        FunctionName="HttpTriggerWriteToQueue1",
+                        Bindings= new Dictionary<string, string>
+                        {
+                           {"httpTrigger", "request"},
+                           {"queue", "$return"}
```

Comment:
So the existing tests were only validating type/name, and not other properties like "connection" etc.? I.e. how do I know for these output bindings you've added that Connection value is making it through?

Reply:
Yes. I updated the tests to validate all attributes now.